src/univariate/continuous/noncentralbeta.jl

```diff
@@ -1,5 +1,7 @@
 """
     NoncentralBeta(α, β, λ)
+
+*Noncentral Beta distribution* with shape parameters `α` and `β` and noncentrality parameter `λ`.
```

Comment:
To be consistent, maybe mention the constraints here as well?

```suggestion
*Noncentral Beta distribution* with shape parameters `α > 0` and `β > 0` and noncentrality parameter `λ >= 0`.
```